web/month.php

```diff
@@ -415,12 +415,12 @@ function cmp3($a, $b) {
         if (!$allowedtobook) {
             // No permission to book this room
             $title = get_string('notallowedbook', 'block_mrbs');
-            echo '<img src="'.$OUTPUT->pix_url('toofaradvance', 'block_mrbs').'" width="10" height="10" border="0" alt="'.$title.'" title="'.$title.'" />';
+            echo '<img src="'.$OUTPUT->image_url('toofaradvance', 'block_mrbs').'" width="10" height="10" border="0" alt="'.$title.'" title="'.$title.'" />';
```

Comment:
I'd rather this was fixed by using $OUTPUT->pix_icon instead of $OUTPUT->image_url.